### PR TITLE
Improvement: DOS-418 - exposed the delay prop on the tippy tooltip

### PR DESCRIPTION
**Pull request by momustafa-causalens**
## Motivation and Context
Exposes the delay prop

## Implementation Description
Added the delay prop

## Any new dependencies Introduced
No

## How Has This Been Tested?
Manually

## PR Checklist:

- [x] I have implemented all requirements? (see JIRA, project documentation).
- [x] I am not affecting someone else's work, If I am, they are included as a reviewer.
- [x] I have added relevant tests (unit, integration or regression).
- [x] I have added comments to all the bits that are hard to follow.
- [x] I have added/updated Documentation.
- [x]  I have updated the appropriate changelog with a line for my changes.

### CODE REVIEW

**Review by krzysztof-causalens — APPROVED** [review]

**Review by patricia-causalens — APPROVED** [review]
Looks good, nitpicky should maybe be NEXT on changelog unless you are planning to release a version straight away?  Not sure what your plan is :)

**Comment by krzysztof-causalens** [comment]
> Looks good, nitpicky should maybe be NEXT on changelog unless you are planning to release a version straight away? Not sure what your plan is :)

Good catch, you'll need another PR to bump up the versions either way, so yeah probably stick to NEXT in this PR.

**Comment by momustafa-causalens** [comment]
> NEXT


> Looks good, nitpicky should maybe be NEXT on changelog unless you are planning to release a version straight away? Not sure what your plan is :)

do i bump the version or will u guys? ive got a small PR waiting on this change

**Comment by krzysztof-causalens** [comment]
I can make a release right after this is merged
